src/index.html

```diff
@@ -0,0 +1,3 @@
+<!DOCTYPE html>
+
+Hello, world!
```

Comment:
add base markup (html, head, body)
you can do this with emmet shortcut (write '!' and press enter)